cloud/amazon/iam_policy.py

```diff
@@ -283,7 +283,7 @@ def main():
       iam_name=dict(default=None, required=False),
       policy_name=dict(default=None, required=True),
       policy_document=dict(default=None, required=False),
-      policy_json=dict(default=None, required=False),
+      policy_json=dict(type='dict', default=None, required=False),
```

Comment:
actually this breaks for those passing in a json string, this should be `type=json` in devel and requires some code changes to backport to 2.1